content/TruenasApps/StableApps/InstallNextCloudMedia.md

```diff
@@ -15,151 +15,284 @@ keywords:
 
-* **appdata** that contains HTML, apps, custom_themes, config, etc.
-* **userdata** that contains the actual files uploaded by the user
-* **pgdata** that contains the database files.
-* **pgbackup** that contains the database backups
+  When adding the **postgres_data** dataset, enter the dataset name and then click **Advanced Options** to show the advanced dataset settings.
+  Scroll down to the **ACL Type** and select **POSIX** from the dropdown list, and then click **Save**.
+  Only the **postgres_data** dataset requires the POSIX ACL type setting.
 
-TrueNAS creates the **ix-apps** dataset in the pool you set as the application pool when you first go to the **Apps** screen.
-This dataset is internally managed, so you cannot use this as the parent when you create the required Nextcloud datasets.
+  {{< trueimage src="/images/SCALE/Apps/SetPostgres_dataACLtoPOSIX.png" alt="Set postgres_data Dataset ACL Type" id="Set postgres_data Dataset ACL Type" >}}
 
-To create the Nextcloud app datasets, go to **Datasets**, select the dataset you want to use as the parent dataset, then click **Add Dataset** to [add a dataset]({{< relref "DatasetsScale.md" >}}).
-In this example, we create the Nextcloud datasets under the root parent dataset **tank**.
+  Click **Set ACL for this dataset** to open the **Edit ACL** screen.
+  Set the **owner** and **group** to **netdata** and click **Apply Owner** and **Apply Group**, and then with that ACL entry highlighted, assign full control permissions before you save the ACL.
+  Click **Save Access Control List**.
 
-Enter **nextcloud** in **Name**, select **Apps** as the **Dataset Preset**.
-Click **Advanced Options** to make any other setting changes you want to make, and click **Save**.
-When prompted, select **Return to Pool List**.
+  {{< trueimage src="/images/SCALE/Apps/AddPostgres_DataACLPermissions.png" alt="Add Nextcloud postgres_data Dataset ACL Permissions" id="Add Nextcloud postgres_data Dataset ACL Permissions" >}}
 
-Next, select the **nextcloud** dataset, click **Add Dataset** to add the first child dataset.
-Enter **appdata** in **Name** and select **Apps** as the **Dataset Preset**.
-Click **Advanced Options** to make any other setting changes you want to make for the dataset, and click **Save**.
+  {{< /expand >}}
 
-Repeat this three more times to add the other three child datasets to the **nextcloud** parent dataset.
-When finished you should have the **nextcloud** parent dataset with four child datasets under it. Our example paths are:
-* */mnt/tank/nextcloud/appdata*
-* */mnt/tank/nextcloud/userdata*
-* */mnt/tank/nextcloud/pgdata*
-* */mnt/tank/nextcloud/pgbackup*
 
-{{< trueimage src="/images/SCALE/Apps/AppsAddNextcloudDatasets.png" alt="Add Nextcloud Storage" id="Add Nextcloud Storage" >}}
+{{< include file="/static/includes/apps/AppBeforeYouBeginNewUser.md" >}}
```

Comment:
Is this intended to be here? Its also on the top. But still feels weird to be here.

Reply:
Yes, this is the instruction for creating a new user. The similar name of the snippet file might be throwing you off. ;-)